streams.json

```diff
@@ -38,20 +42,24 @@
           "repository_type": "ARCHIVE",
           "repository_url": "http://www.antlr2.org/download/antlr-2.7.7.tar.gz",
           "codebase": "",
-          "tag": "2.7.7",
+          "tag": "unknown",
+          "group_id": "antlr",
+          "artifact_id": "antlr",
           "version": "2.7.7",
-          "gav": "antlr",
+          "gav": "antlr:antlr",
           "comment": "Tag is just a guess"
         },
         {
           "component_name": "apache-activemq-artemis",
           "contacts": [],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/rh-messaging/jboss-activemq-artemis",
+          "repository_url": "https://github.com/apache/activemq-artemis.git",
```

Comment:
Wrong.

Reply:
@wolfc Fixed, I added this URL to our dictionary file